validator/validator.c

```diff
@@ -200,6 +200,17 @@ val_init(struct module_env* env, int id)
 		log_err("validator: could not apply configuration settings.");
 		return 0;
 	}
+	if(env->cfg->disable_edns_do) {
+		struct trust_anchor* anchor = anchors_find_any_noninsecure(
+			env->anchors);
+		if(anchor) {
+			char b[257];
```

Comment:
I would prefer the use of a defined valued instead
```suggestion
			char b[LDNS_MAX_DOMAINLEN+2];
```

Reply:
Thanks, committed.